utils/docker-compose/scripts/lib/common.sh

```diff
@@ -16,6 +16,7 @@ export SGX_MODE=${SGX_MODE:=SIM}
 
 # Variables which we allow the caller override ..
 export FABRIC_VERSION=${FABRIC_VERSION:=1.4.3}
+export FPC_VERSION=${FPC_VERSION:=latest}
```

Comment:
probably should be FPC_IMAGE_VERSION

Reply:
why would you think so?  We also say FABRIC_VERSION?